test/server/services/responses.js

```diff
@@ -70,39 +94,92 @@ describe('Responses Service tests', () => {
               this.campaignParams.contact_lists_id = contactList.attributes.id;
               campaignsService.saveNewCampaign(this.campaignParams)
                 .then((campaign) => {
-                  this.callSaveParams.campaign_id = campaign.attributes.id;
-                  contactsService.saveNewContact(this.contactSaveParams)
-                    .then((contact) => {
-                      this.callSaveParams.contact_id = contact.attributes.id;
-                      usersService.saveNewUser(this.userSaveParams)
+                  this.callSaveParams = [];
```

Comment:
this.callSaveParams is set as an empty object above - should that be deleted?

Reply:
yes, I'll do that meow